doc/api/test.md

```diff
@@ -1352,8 +1352,15 @@ test('changes a mock behavior once', (t) => {
   const fn = t.mock.fn(addOne);
```

Comment:
How do I update the:

```
### `ctx.mockImplementationOnce(implementation[, onCall])`
<!-- YAML
added:
  - v19.1.0
  - v18.13.0
-->
```

to have the change and current version this is added?

Reply:
@rluvaton use the `REPLACEME` "keyword":
https://github.com/nodejs/node/blob/78891f09a64f5a2ed82206dd9959dd1ce9951078/doc/contributing/pull-requests.md#L122-L133

Reply:
For `changed` see these example: https://github.com/nodejs/node/blob/78891f09a64f5a2ed82206dd9959dd1ce9951078/doc/api/inspector.md#L422-L427 https://github.com/nodejs/node/blob/78891f09a64f5a2ed82206dd9959dd1ce9951078/doc/api/net.md#L577-L584

Reply:
thanks, done